src/core.jl

```diff
@@ -109,8 +213,23 @@ function check_options(options::NomadOptions)
         end
     end
     (options.max_bb_eval > 0) ? nothing : error("NOMAD.jl error: wrong parameters, max_bb_eval must be strictly positive")
+    (options.eval_queue_sort ∈ EvalSortTypes) ? nothing : error("NOMAD.jl error: wrong parameters, eval_queue_sort must belong to EvalSortTypes, i.e. $(EvalSortTypes)")
+    (options.sgtelib_model_max_eval > 0) ? nothing : error("NOMAD.jl error: wrong parameters, sgtelib_model_max_eval must be strictly positive")
```

Comment:
You can replace all `? nothing :` by a `||`.

Reply:
Done